index.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const methodOverride = require('method-override');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const path = require('path');
+const clucksRouter = require('./routes/clucks/clucksRouter');
+const knex = require('./db/client')
+
+const app = express();
+
+function timeDifference(current, previous) {
```

Comment:
this logic should not live here. index should mostly contain express configurations. suggestion: create `helper/time_calculation` file and move it there.